alt-tab-macos/logic/SystemPermissions.swift

```diff
@@ -4,25 +4,20 @@ import Cocoa
 // macOS has some privacy restrictions. The user needs to grant certain permissions, app by app, in System Preferences > Security & Privacy > Privacy
 class SystemPermissions {
     // macOS 10.9+
-    static func ensureAccessibilityCheckboxIsChecked() {
+    static func checkAccessibility() -> Bool {
         if !AXIsProcessTrustedWithOptions(["AXTrustedCheckOptionPrompt": true] as CFDictionary) {
-            debugPrint("Before using this app, you need to give permission in System Preferences > Security & Privacy > Privacy > Accessibility.",
-                    "Please authorize and re-launch.",
-                    "See https://help.rescuetime.com/article/59-how-do-i-enable-accessibility-permissions-on-mac-osx",
-                    separator: "\n")
-            NSApp.terminate(self)
+            return false
         }
+        return true
     }
 
     // macOS 10.15+
-    static func ensureScreenRecordingCheckboxIsChecked() {
+    static func checkScreenshot() -> Bool {
```

Comment:
I wonder if the private API I switched to in v2 still need this checkbox to be checked. I don't have a Catalina machine to test it though

Reply:
On my macOS 10.15.2, if the privilege isn't allowed, the screen previews don't show up, and the privilege prompt shows up behind the selector panel. I think that's still better than preventing the app from starting at all :)